wavefront/server/modules/plugins_module/plugins_module/controllers/cloud_storage_controller.py

```
@@ -0,0 +1,47 @@
+            bucket_name=bucket_name,
+            key=key,
+            type='GET',
+            expiresIn=expires_in,
+        )
+        return JSONResponse(
+            status_code=status.HTTP_200_OK,
+            content=response_formatter.buildSuccessResponse(
+                {'resource_url': presigned_url}
+            ),
+        )
+    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)